demos/projects/ESPRESSIF/aziotkit/components/azure-iot-kit-sensors/src/led.c

```diff
@@ -0,0 +1,25 @@
+#include "led.h"
+// #include "esp_rom_gpio.h"
+#include "hal/gpio_types.h"
+#include "driver/gpio.h"
```

Comment:
are other includes required? like #include "hal/gpio_types.h" and #include "esp32/rom/gpio.h" ?